src/__tests__/string.test.ts

```diff
@@ -101,13 +101,11 @@ test("more email validations", () => {
 
 test("url validations", () => {
   const url = z.string().url();
-  try {
-    url.parse("http://google.com");
-    url.parse("https://google.com/asdf?asdf=ljk3lk4&asdf=234#asdf");
-    expect(() => url.parse("asdf")).toThrow();
-    expect(() => url.parse("https:/")).toThrow();
-    expect(() => url.parse("asdfj@lkjsdf.com")).toThrow();
-  } catch (err) {}
```

Comment:
My original PR #2045 copied from this test, which had this empty `catch`, so I though it would also fix this in the follow up.